app/Models/Wallet.php

```diff
@@ -28,4 +30,6 @@ class Wallet extends Model
         'currency',
         'balance',
     ];
+
+    protected $appends = ['last_synced_at'];
```

Comment:
You forgot to add the method to the model
`Call to undefined method App\Models\Wallet::getLastSyncedAtAttribute()`